src/model/modifier/__tests__/__snapshots__/AtomicBlockUtils-test.js.snap

```diff
@@ -1,1306 +1,5573 @@
+        "style": Array [],
+      },
+    ],
+    "data": Object {},
+    "depth": 0,
+    "key": "key0",
+    "text": " ",
+    "type": "atomic",
+  },
+  Object {
+    "characterList": Array [],
+    "data": Object {},
+    "depth": 0,
+    "key": "key4",
+    "text": "",
+    "type": "unstyled",
+  },
+  Object {
+    "characterList": Array [
+      Object {
+        "entity": "1",
+        "style": Array [
+          "BOLD",
+        ],
+      },
+      Object {
+        "entity": "1",
+        "style": Array [
+          "BOLD",
+        ],
+      },
+      Object {
+        "entity": "1",
+        "style": Array [
+          "BOLD",
+        ],
+      },
+      Object {
+        "entity": "1",
+        "style": Array [
+          "BOLD",
+        ],
+      },
+      Object {
+        "entity": "1",
+        "style": Array [
+          "BOLD",
+        ],
+      },
+    ],
+    "data": Object {},
+    "depth": 0,
     "key": "b",
     "text": "Bravo",
     "type": "unordered-list-item",
   },
-  "c": Object {
+  Object {
+    "characterList": Array [
+      Object {
+        "entity": null,
+        "style": Array [],
+      },
+      Object {
+        "entity": null,
+        "style": Array [],
+      },
+      Object {
+        "entity": null,
+        "style": Array [],
+      },
+      Object {
+        "entity": null,
+        "style": Array [],
+      },
+    ],
+    "data": Object {},
+    "depth": 0,
     "key": "c",
     "text": "Test",
     "type": "code-block",
   },
-  "d": Object {
+  Object {
+    "characterList": Array [],
+    "data": Object {},
+    "depth": 0,
     "key": "d",
     "text": "",
     "type": "code-block",
   },
-  "e": Object {
+  Object {
+    "characterList": Array [],
+    "data": Object {},
+    "depth": 0,
     "key": "e",
     "text": "",
     "type": "code-block",
   },
-  "f": Object {
+  Object {
+    "characterList": Array [
+      Object {
+        "entity": null,
+        "style": Array [
+          "ITALIC",
+        ],
+      },
+      Object {
+        "entity": null,
+        "style": Array [
+          "ITALIC",
+        ],
+      },
+      Object {
+        "entity": null,
+        "style": Array [
+          "ITALIC",
+        ],
+      },
+      Object {
+        "entity": null,
+        "style": Array [
+          "ITALIC",
+        ],
+      },
+      Object {
+        "entity": null,
+        "style": Array [
+          "ITALIC",
+        ],
+      },
+      Object {
+        "entity": null,
+        "style": Array [
+          "ITALIC",
+        ],
+      },
+      Object {
+        "entity": null,
+        "style": Array [
+          "ITALIC",
+        ],
+      },
+    ],
+    "data": Object {},
+    "depth": 0,
     "key": "f",
     "text": "Charlie",
     "type": "blockquote",
   },
-  "key11": Object {
-    "key": "key11",
-    "text": "",
-    "type": "unstyled",
-  },
-  "key8": Object {
-    "key": "key8",
-    "text": " ",
-    "type": "atomic",
-  },
-}
```

Comment:
I am concerned that the ordering of the blocks in the `must insert atomic after a block with collapsed selection 1` snapshot has changed. Is that expected?

It was `a, b, c, d, e, f, key11, key8`.
In the updated snapshot it is `a, key0, key4, b, c, d, e, f`.

Reply:
The previous order was incorrect, it was like that due to when we convert ordered map to javascript the order is alphanumeric not order of insertion, when we changed the assertions to use indexed sequence we guaranteed that the ordering of the snapshot was correct.

Reply:
I have just double checked, and if I update the assertion method to how did before

```javascript
      .getBlockMap()
      .map(block => ({
        key: block.getKey(),
        type: block.getType(),
        text: block.getText(),
      }))
      .toJS(),
```
The snapshot is the same.

so the reason why before "key0" and "key4" were in the end is because "k" is after "f" and when we convert from orderedMap with the .toJS() it does not preserve insertion order, that why we update the assertions to use ` .toIndexedSeq()` before converting to JS:

```
      .getBlockMap()
      .toIndexedSeq()
      .toJS(),

```